Sources/Core/AWSSDKIdentity/Sources/InternalAWSSTS/AuthSchemeResolver.swift

```diff
@@ -8,12 +8,10 @@
 // Code generated by smithy-swift-codegen. DO NOT EDIT!
```

Comment:
Codegen change in this PR removes internal client init codegen for auth scheme resolvers of internal clients. It's not necessary + codegen'ing it creates circular dependency between internal client targets.